src/features/identityCheck/pages/identification/educonnect/IdentityCheckEduConnectForm.web.test.tsx

```diff
@@ -26,4 +26,12 @@ describe('<IdentityCheckEduConnectForm />', () => {
 
     expect(mockOpenEduConnectTab).toHaveBeenCalledTimes(1)
   })
+
+  describe('Accessibility', () => {
+    it('should not have basic accessibility issues', async () => {
+      const { container } = render(<IdentityCheckEduConnectForm />)
+      const results = await checkAccessibilityFor(container)
+      expect(results).toHaveNoViolations()
+    })
+  })
```

Comment:
On utilise un bouton au lieu d'un `ExternalTouchableLink` dans `IdentityCheckEduConnectForm.web.tsx`
Le test d'accessibilité ne peut pas détecter s'il faut utiliser un bouton ou un lien, mais ça serait top de faire cette modif (dans cette PR ou une autre, comme tu veux)

Reply:
Je créé un ticket

Reply:
Je crois que nous pouvons créer [nos propres règles d'accessibilité avec la lib](https://github.com/nickcolley/jest-axe#setting-custom-rules-and-checks), mais a voir si c'est utile / faisable pour certaines règles d'accessibilité propre a notre repo.